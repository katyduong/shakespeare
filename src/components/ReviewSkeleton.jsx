import React from 'react';
import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';

const ReviewSkeleton = () => (
  <SkeletonTheme color="#f5f5f5" highlightColor="#e9ecef">
    <div className="jumbotron p-3 p-md-5 rounded">
      <div className="px-0">
        <h1 className="display-4">
          <Skeleton />
        </h1>
        <p>
          <Skeleton />
        </p>
        <p className="lead my-3">
          <Skeleton count={2} />
        </p>
      </div>
    </div>
  </SkeletonTheme>
);

export default ReviewSkeleton;
