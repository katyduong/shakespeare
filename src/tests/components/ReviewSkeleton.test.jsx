import React from 'react';
import renderer from 'react-test-renderer';
import ReviewSkeleton from '../../components/ReviewSkeleton';

test('renders ReviewSkeleton', () => {
  const tree = renderer.create(<ReviewSkeleton />).toJSON();
  expect(tree).toMatchSnapshot();
});
