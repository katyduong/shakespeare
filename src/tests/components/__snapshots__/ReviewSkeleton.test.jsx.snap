// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ReviewSkeleton 1`] = `
<div
  className="css-1r10mqi-SkeletonTheme"
>
  <div
    className="jumbotron p-3 p-md-5 rounded"
  >
    <div
      className="px-0"
    >
      <h1
        className="display-4"
      >
        <span>
          <span
            className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
            style={Object {}}
          >
            ‌
          </span>
        </span>
      </h1>
      <p>
        <span>
          <span
            className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
            style={Object {}}
          >
            ‌
          </span>
        </span>
      </p>
      <p
        className="lead my-3"
      >
        <span>
          <span
            className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
            style={Object {}}
          >
            ‌
          </span>
          <span
            className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
            style={Object {}}
          >
            ‌
          </span>
        </span>
      </p>
    </div>
  </div>
</div>
`;
